src/recipe/parser/build.rs

```diff
@@ -104,16 +104,16 @@ impl TryConvertNode<Build> for RenderedMappingNode {
                 "noarch" => {
                     build.noarch = value.try_convert(key_str)?;
                 }
-                "entry_points" => {
+                "python" => {
                     if let Some(NoArchKind::Generic) = build.noarch.kind() {
                         return Err(_partialerror!(
                             *key.span(),
                             ErrorKind::Other,
-                            label = "`entry_points` are only allowed for `python` noarch packages"
+                            label = "`python` is only allowed for `python` noarch packages"
```

Comment:
I think this is not true. Even for `entry_points` it was already wrong (we added entry_points to the `rerun` recipe and it's not noarch).

Some of the other options (`skip_pyc_compilation` are also in use for non-noarch packages I believe).

Reply:
Ok! I removed it.